src/components/organisms/OurSupportersSection/OurSupportersSection2.js

```diff
@@ -0,0 +1,293 @@
+            </div>
+            <div className="flex mx-1">
+              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter14} ImageStyleProps="" />
+            </div>
+            <div className="flex mx-1">
+              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter15} ImageStyleProps="" />
+            </div>
+            <div className="flex mx-1">
+              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter16} ImageStyleProps="" />
+            </div>
+          </div>
+        </div>
+
+        <div>
+          <div className="w-full flex justify-center" >
+            <div className="flex mx-1">
+              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter17} ImageStyleProps="" />
+            </div>
+            <div className="flex mx-1">
+              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter18} ImageStyleProps="" />
+            </div>
+            <div className="flex mx-1">
+              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter19} ImageStyleProps="" />
+            </div>
+            <div className="flex mx-1">
+              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter20} ImageStyleProps="" />
+            </div>
+          </div>
+        </div>
+
+        <div>
+          <div className="w-full flex justify-center" >
+            <div className="flex mx-1">
+              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter21} ImageStyleProps="" />
+            </div>
+            <div className="flex mx-1">
+              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter22} ImageStyleProps="" />
+            </div>
+            <div className="flex mx-1">
+              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter23} ImageStyleProps="" />
+            </div>
+            <div className="flex mx-1">
+              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter24} ImageStyleProps="" />
+            </div>
+          </div>
+        </div>
+
+        <div>
+          <div className="w-full flex justify-center" >
+            <div className="flex mx-1">
+              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter25} ImageStyleProps="" />
+            </div>
+            <div className="flex mx-1">
+              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter26} ImageStyleProps="" />
+            </div>
+            <div className="flex mx-1">
+              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter27} ImageStyleProps="" />
+            </div>
+          </div>
+        </div>
+
+      </div> 
+
+
+    {/* Desktop view */}
+      <div className="max-w-[1440px] w-full hidden md:block" >
+        <div className="mb-40"></div>
+        
+        {/* Supporters by row */}
+        <div>
+          <div className="w-full flex justify-center my-14" >
+            <div className="flex mx-10">
```

Comment:
Hm, if styling is consistent for each image and container in the mobile and desktop view, instead of hardcoding 10+ sets of divs with the same styling, are you willing to refactor it (ie. map() the images and render it using a reusable component) to reduce code duplication and improve readability and maintainability?

Reply:
I definitely thought about that while write that page. I will make the changes!